build.assets/Makefile

```diff
@@ -27,7 +27,7 @@ GOLANG_VERSION ?= go1.19.4
 
 NODE_VERSION ?= 16.18.1
 
-RUST_VERSION ?= 1.64.0 # don't bump this without checking GLIBC compatibility
+RUST_VERSION ?= 1.66.0 # don't bump this without checking GLIBC compatibility
```

Comment:
@zmb3 Does it still make sense to check the GLIBC version now when we build on CentOS 7 with GLIBC 2.17 anyways?

Reply:
Na, we can safely remove this comment since the only way it can fail now is if Rust itself requires a newer version of GLIBC, and the build on CentOS 7 would fail in this case.

Reply:
The comment is gone 😄